README.md

```diff
@@ -16,15 +16,17 @@ permissions and limitations under the License. See accompanying
 LICENSE file.
 -->
 
-Yahoo! Cloud System Benchmark (YCSB)
+YCSB
 ====================================
 [![Build Status](https://travis-ci.org/brianfrankcooper/YCSB.png?branch=master)](https://travis-ci.org/brianfrankcooper/YCSB)
 
+
+
 Links
 -----
-http://wiki.github.com/brianfrankcooper/YCSB/  
-https://labs.yahoo.com/news/yahoo-cloud-serving-benchmark/
-ycsb-users@yahoogroups.com  
+* To get here, use http://ycsb.site
```

Comment:
Nit: should this be `https://ycsb.site`? Looks like it worked on https when I tried it.

Reply:
I didn't think I set up TLS, so that's interesting. 😹

Reply:
so it is. I'll amend this before pushing. thanks!